src/Interfaces/LazySet.jl

```diff
@@ -878,3 +879,93 @@ true
 ```
 """
 isconvextype(X::Type{<:LazySet}) = false
+
+"""
+    area(X::LazySet{N}) where {N}
+
+Compute the area of a set.
+
+### Input
+
+- `X` -- set
+
+### Output
+
+A number representing the area of `X`.
+"""
+function area(X::LazySet{N}) where {N}
+    if dim(X) == 2
+        return area_2D(X)
+    else
+        throw(ArgumentError("the area of a $(dim(X))-dimensional set is not " *
+                            "implemented yet"))
```

Comment:
Can you add some definition of the area of a non-2D set? I would just define it for 2D sets. Otherwise maybe it should be called `volume` or `surface` if it generalizes.

Reply:
Yes, i had in mind the (surface) area.

Reply:
What change do you propose? The area *is* defined only for 2D sets in this PR. Or you suggest to change the error message to something else, like "the area function only applies to two-dimensional sets, but the given set is $(dim(X))-dimensional"?

Reply:
Yes, I would change the error message to the one you suggested. We can still have this proxy method `area` but then if we want to add more methods, we have to call them `area2D` (just saying).

Reply:
ah, i see. we can rename `area -> surface` (any dimension, in particular uses `area` for 2D) and `area_2D -> area` (only 2D). but this wouldn't be breaking, as `surface` is not available yet, we can do this at a later moment.